README.rst

```diff
@@ -688,7 +688,7 @@ snippet:
 
     some_func()
 
-NaÃƒÆ’Ã†â€™Ãƒâ€šÃ‚Â¯vely evaluating such preconditions and postconditions would result in endless recursions. Therefore, icontract
+NaÃƒÆ’Ã†â€™Ãƒâ€ Ã¢â‚¬â„¢ÃƒÆ’Ã¢â‚¬Å¡Ãƒâ€šÃ‚Â¯vely evaluating such preconditions and postconditions would result in endless recursions. Therefore, icontract
```

Comment:
That looks broken.

Reply:
Thanks a lot for spotting it! This got broken in a previous pull request.